VAS.Tests/Services/TestPlaylistController.cs

```
@@ -38,13 +39,21 @@ public class TestPlaylistController
 		Mock<IGUIToolkit> mockGuiToolkit;
 		Mock<IPlayerViewModel> mockPlayerController;
 		Mock<IDialogs> mockDiaklogs;
+		Mock<IStorageManager> storageManagerMock;
+		Mock<IStorage> storageMock;
 		PlaylistController controller;
 
 		[TestFixtureSetUp ()]
 		public void FixtureSetup ()
 		{
 			mockPlayerController = new Mock<IPlayerViewModel> ();
 			mockGuiToolkit = new Mock<IGUIToolkit> ();
+
+			storageManagerMock = new Mock<IStorageManager> ();
+			storageManagerMock.SetupAllProperties ();
+			storageMock = new Mock<IStorage> ();
+			storageManagerMock.Object.ActiveDB = storageMock.Object;
+			App.Current.DatabaseManager = storageManagerMock.Object;
```

Playlist are saved when are created or modified, so we need to mock Database in test. If not, TestAddSamePlaylist and TestNewPlaylist fails